### PR TITLE
update index.md

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
add one word "be".

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2755)

### CODE REVIEW

File: docs/tools/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Use Helm to:
 
 ##### Kompose 
 
-[Kompose](https://github.com/kubernetes-incubator/kompose) is a tool to help users familiar with Docker Compose
+[Kompose](https://github.com/kubernetes-incubator/kompose) is a tool to help users be familiar with Docker Compose
```
Comment (janetkuo) [inline docs/tools/index.md]: The original wording isn't wrong. It helps users that are already familiar with Docker Compose, not helps users to become familiar with Docker Compose. 

Maybe we can rephrase this to something like "...to help Docker Compose users move to Kubernetes"?

Comment (techtonik) [inline docs/tools/index.md]: ...is a tool for users, who are familiar...

**Comment by janetkuo** [comment]
Thanks, LGTM. Please rebase and squash commits.
